lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -134,8 +134,8 @@ class OffscreenImages extends ByteEfficiencyAudit {
       });
 
       const headings = [
-        {key: 'preview', itemType: 'thumbnail', text: ''},
-        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'preview', itemType: 'thumbnail', text: '', styleAs: 'preview'},
+        {key: 'url', itemType: 'url', text: 'URL', styleAs: 'url'},
```

Comment:
ah, `styleClass` does make it clearer. Is there a more general name we can use over `url` to avoid confusion there too? (`preview` is also pretty non-descriptive, but I don't know if we can do better)